### PR TITLE
Add org.nsnam.NetAnim

### PR DESCRIPTION
**Pull request by marc0x1**
### Please confirm your submission meets all the criteria

- [x] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [x] My pull request follows the instructions at [App Submission][submission].
- [x] I have [built][build] and tested the submission locally.
- [x] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [x] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [x] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. https://gitlab.com/nsnam/netanim/-/issues/8
- [ ] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [ ] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

[reqs]: https://docs.flathub.org/docs/for-app-authors/requirements
[maint]: https://docs.flathub.org/docs/for-app-authors/maintanance
[submission]: https://docs.flathub.org/docs/for-app-authors/submission
[build]: https://docs.flathub.org/docs/for-app-authors/submission/#before-submission

### CODE REVIEW

File: org.nsnam.NetAnim.yaml [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+    sources:
+      - type: git
+        url: https://gitlab.com/nsnam/netanim
+        tag: netanim-3.109
+        x-checker-data:
+          type: git
+          tag-pattern: ^netanim-([\d.]+)$
+          
+      - type: file
+        path: CMakeLists.txt
```
Comment (hfiguiere) [inline org.nsnam.NetAnim.yaml:24]: why is this here? It doesn't have a build system?

File: org.nsnam.NetAnim.metainfo.xml
Comment (hfiguiere) [inline org.nsnam.NetAnim.metainfo.xml:1]: this file should be submitted upstream

File: org.nsnam.NetAnim.desktop
Comment (hfiguiere) [inline org.nsnam.NetAnim.desktop:1]: this file should be submitted upstream. A desktop app without a .desktop, it's a bug

File: icon.png
Comment (hfiguiere) [inline icon.png:1]: icon should also be submitted upstream to be installed at the same time as the .desktop.

File: org.nsnam.NetAnim.metainfo.xml [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+  </supports>
+  <recommends>
+    <control>keyboard</control>
+  </recommends>
+  
+  <description>
+    <p>
+      NetAnim is an offline animator based on the Qt toolkit. It animates a previously executed simulation using an XML trace file generated during a simulation.
+    </p>
+    <p>This wrapper is not verified by, affiliated with, or supported by the NetAnim Team.</p>
```
Comment (hfiguiere) [inline org.nsnam.NetAnim.metainfo.xml:33]: usually this disclaimer is only used for proprietary software.

File: org.nsnam.NetAnim.yaml (hunk 1 above)
Comment (marc0x1) [reply org.nsnam.NetAnim.yaml:24]: upstream there is an open mr to add it. When it is ready I will remove it

Comment (hfiguiere) [reply org.nsnam.NetAnim.yaml:24]: then if it's not read why using it here? qmake no good?

File: CMakeLists.txt
Comment (barthalion) [inline CMakeLists.txt:1]: This probably shouldn't be here?

**Comment by barthalion** [comment]
bot, build org.nsnam.NetAnim

File: org.nsnam.NetAnim.desktop
Comment (marc0x1) [reply org.nsnam.NetAnim.desktop:1]: Will be sent when upstream responds. Is it blocking for approval?

File: org.nsnam.NetAnim.yaml [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+command: netanim
+finish-args:
+  - --socket=fallback-x11
+  - --share=ipc
+  - --socket=wayland
+  - --device=dri
+modules:
+  - name: netanim
+    buildsystem: qmake
+    build-commands:
```
Comment (hfiguiere) [inline org.nsnam.NetAnim.yaml:14]:
```suggestion
    post-install:
```

File: org.nsnam.NetAnim.yaml [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+      - type: file
+        path: org.nsnam.NetAnim.metainfo.xml
+      - type: file
+        path: icon.png
+      - type: git
+        url: https://gitlab.com/nsnam/netanim
+        tag: netanim-3.109
+        x-checker-data:
+          type: git
+          tag-pattern: ^netanim-([\d.]+)$
```
Comment (hfiguiere) [inline org.nsnam.NetAnim.yaml:31]:
```suggestion
      - type: git
        url: https://gitlab.com/nsnam/netanim
        tag: netanim-3.109
        x-checker-data:
          type: git
          tag-pattern: ^netanim-([\d.]+)$
      - type: file
        path: org.nsnam.NetAnim.desktop
      - type: file
        path: org.nsnam.NetAnim.metainfo.xml
      - type: file
        path: icon.png
```

**Comment by bbhtt** [comment]
bot, build org.nsnam.NetAnim

**Comment by barthalion** [comment]
bot, build org.nsnam.NetAnim
